src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -555,7 +634,14 @@ public void Shutdown()
 
         public PackageId GetManifestPackageId(ManifestId manifestId, SdkFeatureBand featureBand)
```

Comment:
This is misleading, I'd suggest having two separate methods for manifest and workload set package, even though there similar - it will make it much easier to understand the code.

Reply:
I'll admit you're right that it's a little misleading. I'm trying to balance making things clearer vs. reusing more code—since this method is called in the middle of where I need it, I'd have to either put a random if/else in the middle of that method (that is, move this up one stack frame) or copy most of the method into a separate method just for workload sets. I'd rather not lose any updates that come in the future, which is why I architected it this way. If you feel strongly, though, I can change that.

Reply:
In GetManifestPackageDownloadsAsync and UpdateManifestWithVersionAsync, we shouldn't know whether it's a workload set manifest ID or something else.